src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rand = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```

плохая ппрактика использовать сокращения в названиях переменых